visual-tests/text-wrapping-hyphen/text-wrapping-hyphen-test.cpp

```diff
@@ -71,6 +73,8 @@ class TextWrappingTest : public VisualTest {
     mTextLabel.SetProperty(TextLabel::Property::POINT_SIZE, 200);
     mTextLabel.SetProperty(TextLabel::Property::ELLIPSIS, false);
     mTextLabel.SetProperty(TextLabel::Property::MULTI_LINE, true);
+    mTextLabel.SetProperty(DevelTextLabel::Property::REMOVE_FRONT_INSET, true);
```

Comment:
Please add some comment as TODO note, here, and all other inset cases.
```suggestion
    // TODO : Since 2.3.32, we change the default value of REMOVE_FRONT_INSET and REMOVE_BACK_INSET as false.
    //              We need to change expect result of those tests in future
    mTextLabel.SetProperty(DevelTextLabel::Property::REMOVE_FRONT_INSET, true);
```

Reply:
Done